docs/pages/kubernetes-access/helm/guides/custom.mdx

```diff
@@ -0,0 +1,273 @@
+---
+title: Running a Teleport cluster with a custom configuration using Helm
+description: Install and configure a Teleport cluster with a custom configuration using Helm
+---
+
+In this guide, we'll go through how to set up a Teleport cluster in Kubernetes using a custom [`teleport.yaml`](../../../config-reference.mdx)
+config file using Teleport Helm charts.
+
+This setup can be useful when you already have an existing Teleport cluster and would like to start running it in Kubernetes, or when
+migrating your setup from an older version of the Helm charts.
+
+### Install Helm
```

Comment:
Seems like prerequisites are full duplicate for all sections as well, consider using `(\! include.mdx \!)
` directive:

https://goteleport.com/docs/docs/best-practices/#includes